tests/test_packages/test_connections/test_oef/test_communication.py

```diff
@@ -1011,16 +1007,18 @@ def _node(self):
         message=msg_bytes,
     )
     await oef_connection.send(envelope)
+    search_result = await oef_connection.receive()
+    assert isinstance(search_result, Envelope)
+    await asyncio.sleep(2.0)
```

Comment:
the main contribution is to add a delay between the `send`/`receive` and the disconnection.
Indeed, I noticed the disconnection didn't complete due to `AsyncioCore.stop` that in some case never terminated.